mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature

```diff
@@ -2333,4 +2333,86 @@ Feature: Loans tests
+    When method GET
+    Then status 200
+    * def currentCirculationRulesAsText = response.rulesAsText
+
+    # post new loan, request, notice, overdue-fine and lost-item-fee policies
+    * def loanPolicyEntityRequest = read('classpath:vega/mod-circulation/features/samples/policies/loan-policy-entity-request.json')
+    * loanPolicyEntityRequest.id = newLoanPolicyId
+    * loanPolicyEntityRequest.name = 'Loan Policy for FAT-991'
+    Given path 'loan-policy-storage/loan-policies'
+    And request loanPolicyEntityRequest
+    When method POST
+    Then status 201
+
+    * def requestPolicyEntityRequest = read('classpath:vega/mod-circulation/features/samples/policies/request-policy-entity-request.json')
+    * requestPolicyEntityRequest.id = newRequestPolicyId
+    * requestPolicyEntityRequest.name = 'Request Policy for FAT-991'
+    * requestPolicyEntityRequest.requestTypes = ["Hold", "Page", "Recall"]
+    Given path 'request-policy-storage/request-policies'
+    And request requestPolicyEntityRequest
+    When method POST
+    Then status 201
+
+    * def patronNoticePolicyEntityRequest = read('classpath:vega/mod-circulation/features/samples/policies/patron-notice-policy-entity-request.json')
+    * patronNoticePolicyEntityRequest.id = newNoticePolicyId
+    * patronNoticePolicyEntityRequest.name = 'Patron Notice Policy for FAT-991'
+    Given path 'patron-notice-policy-storage/patron-notice-policies'
+    And request patronNoticePolicyEntityRequest
+    When method POST
+    Then status 201
+
+    * def overdueFinePolicyEntityRequest = read('classpath:vega/mod-circulation/features/samples/policies/overdue-fine-policy-entity-request.json')
+    * overdueFinePolicyEntityRequest.id = newOverdueFinePolicyId
+    * overdueFinePolicyEntityRequest.name = 'Overdue Fine Policy for FAT-991'
+    Given path 'overdue-fines-policies'
+    And request overdueFinePolicyEntityRequest
+    When method POST
+    Then status 201
+
+    * def lostItemFeePolicyEntityRequest = read('classpath:vega/mod-circulation/features/samples/policies/lost-item-fee-policy-entity-request.json')
+    * lostItemFeePolicyEntityRequest.id = newLostItemFeePolicyId
+    * lostItemFeePolicyEntityRequest.name = 'Lost Item Fee Policy for FAT-991'
+    Given path 'lost-item-fees-policies'
+    And request lostItemFeePolicyEntityRequest
+    When method POST
+    Then status 201
+
+    # verify that loanPolicy has been added successfully (it is just for testing)
+    Given path 'loan-policy-storage', 'loan-policies', newLoanPolicyId
+    When method GET
+    Then status 200
+
+    # enter new circulation rule in the circulation editor
+    * def rules = 'priority: t, s, c, b, a, m, g\nfallback-policy: l '+newLoanPolicyId+' r '+newRequestPolicyId+' n '+newNoticePolicyId+' o '+newOverdueFinePolicyId+' i '+newLostItemFeePolicyId
+    * def rulesEntityRequest = { "rulesAsText" : "#(rules)" }
+    Given path 'circulation', 'rules'
+    And request rulesEntityRequest
+    When method PUT
```

Comment:
When I try to PUT the "rules" It is returning an error. 
But If I PUT this "rules" on ' circulation-rules-storage' endpoint, then policies are being applied correctly so I think there is no error in formatting.
<img width="629" alt="error message" src="https://user-images.githubusercontent.com/112848437/209206728-698a59e9-8021-4ed0-9119-f040b00587a8.png">